docs/index.html

```diff
@@ -35,7 +35,7 @@ <h1>OmegaT User Manual</h1>
 
     <tr>
         <td><a href="en/index.html">English</a></td>
-        <td>(<span class="out-of-date">4.3.1</span>)</td>
+        <td>(<span class="up-to-date">5.8.0</span>)</td>
```

Comment:
Please don't touch a document.

Reply:
OK but this is not really made by me: the problem is that build script always updates this file.
In DGT I had totally removed this file from the Git for this reason: maybe you have a better solution?